tests/dogtag/pytest-ansible/.gitlab-ci.yml

```diff
@@ -1,4 +1,5 @@
-image: bbhavsar/dogtagpki-test-essentials-container
+#image: bbhavsar/dogtagpki-test-essentials-container
+image: docker-registry.upshift.redhat.com/pki/dogtagpki-test-essentials-container
```

Comment:
Is this image available publicly?

Reply:
No, its in Red Hat registry

Reply:
Does the image contain any sensitive information? If not, could we put it https://quay.io/organization/dogtagpki ?

Reply:
Actually yes, its has information pertaining to certs/keytab files for authenticating to Beaker